postgres/init.sls

```diff
@@ -132,8 +134,17 @@ postgresql-tablespace-{{ name }}:
     - name: {{ name }}
     - directory: {{ tblspace.directory }}
     - user: {{ tblspace.get('runas', postgres.user) }}
-{% if tblspace.get('user') %}
-    - db_user: {{ tblspace.user }}
+{% if tblspace.get('db_user') %}
+    - db_user: {{ tblspace.db_user }}
+{% endif %}
+{% if tblspace.get('db_password') %}
```

Comment:
Nice additions. Specifying a connections user without password, hosts and the like doesnt make much sense...